### PR TITLE
Proper creation of trend-chart-pane

### LINKED ISSUE #34: Can't create trend chart
Following my pursuits to using ActivityLog2 to determine my CP, W', etc I tried creating a trend chart of best avg as mentioned in `critical-power.md`. However I am getting:

```
instantiate: no argument for required init variable
  init variable name: trend-chart-class
  instantiated class name: trend-chart-pane%
  context...:
   /home/pmatos/installs/racket-6.12/collects/racket/private/class-internal.rkt:4610:0: obj-error35
   /home/pmatos/installs/ActivityLog2-rel-1.5.0/rkt/view-trends.rkt:186:2
   /home/pmatos/installs/racket-6.12/collects/racket/private/class-internal.rkt:3553:0: continue-make-object
   /home/pmatos/installs/racket-6.12/collects/racket/private/class-internal.rkt:3507:0: do-make-object
   /home/pmatos/installs/ActivityLog2-rel-1.5.0/rkt/view-trends.rkt:340:4: on-new-chart
   /home/pmatos/installs/racket-6.12/share/pkgs/gui-lib/mred/private/wx/common/queue.rkt:428:6
   /home/pmatos/installs/racket-6.12/share/pkgs/gui-lib/mred/private/wx/common/queue.rkt:479:32
   /home/pmatos/installs/racket-6.12/share/pkgs/gui-lib/mred/private/wx/common/queue.rkt:627:3
```

### PR DESCRIPTION
**Pull request by pmatos**
Fixes #34.

### CODE REVIEW

File: rkt/trend-charts/view-trends.rkt [hunk 1]
```diff
@@ -329,27 +330,28 @@
...
           (for ([chart-data (in-list data)])
             (match-define (list chart-tag restore-data) chart-data)
             (make-trends-chart chart-tag restore-data))
           (switch-tabs 0))))
-          
+
     (define (on-new-chart)
       (let ((ct (send (new new-trend-chart-dialog%) show-dialog parent)))
         (when ct
           (let ((pane (let ((tc (new (tdecl-class ct) [database database])))
```
Comment (alex-hhh) [inline rkt/trend-charts/view-trends.rkt]: The `tc` instance is not used anymore, so the `let` binding can be removed.

**Comment by alex-hhh** [comment]
I have no idea why the build is failing (I restarted a build on master and that succeeds).  I will merge this pull request once you remove the `let` form and see if it fails on master as well.

**Comment by pmatos** [comment]
Removed the `let` binding form.

**Comment by pmatos** [comment]
Building might be failing because of this line:
```
openssl aes-256-cbc -d -k $TESTDBPW -in $file_name -out $file_name2
```
in the download test db script. I think `$TESTDBPW` is not being set anywhere.

**Comment by alex-hhh** [comment]
Yes, I figured it out.  The password is set by Travis, but it will only be set on builds from the master repository, otherwise, passwords can be stolen.  I will need to account for this in the download script.  

This feature is used to download some of my personal databases to perform database upgrade tests.  They are encrypted, because I don't want to publish my personal training data.

**Comment by pmatos** [comment]
Makes sense. How hard would it be to create random fit files to populate the database, or even to have a more realistic scenario, randomly scrambling the values in your database?

**Comment by alex-hhh** [comment]
These tests take an existing database and run the upgrade scripts on it -- they make sure that old databases can be upgraded to new ones.  It is hard to create "fresh" old databases, so I just took some old backups for my databases and removed GPS info plus other stuff from these. I decided to encrypt them both as a fun exercise (this is a hobby project, after all) and to increase my privacy.

There are separate tests in "tests/db-tests.rkt" which test that a database can be created and activities imported into it, thee are some FIT files in the test-data folder for this purpose.

**Pull request merged**